test_assets/NormalsTextureBiasAndScale/README.md

```diff
@@ -0,0 +1,149 @@
+# Normals Texture Bias And Scale
+
+This test file, NormalsTextureBiasAndScale.usda, is for testing how well a USD display application interprets the bias and scale inputs for normal map textures. Each cube uses a different normal map texture, combined with different bias and scale values applied to that texture, to give what should be an identical result.
+
+The usdview program from the [USD Toolset](https://graphics.pixar.com/usd/release/toolset.html) includes a basic hydra GL rasterizing renderer. It's about as basic a render you can make, but it's also the standard, in that it's the renderer Pixar provides.
+
+It is possible to build [usdview from scratch](https://graphics.pixar.com/usd/release/toolset.html), but in that way lies madness (at least for me). Happily, [NVIDIA's Omniverse Launcher](https://www.nvidia.com/en-us/omniverse/) provides a pre-built USDView that I can simply install and run. I tested with version 0.22.8.
```

Comment:
Oh, good to know, I'll check this out. I really want to avoid any marketing-sounding stuff here (but am personally happy USDView, as NVIDIA renames it, is available).

Reply:
Hmm, I can't figure out what the Houdini usdview.bat file wants. Do you know? I got far enough in to crash it badly.

I put in:

`C:\Program Files\Side Effects Software\Houdini 19.5.303\bin>usdview.bat C:\Users\ehaines\Documents\_documents\Github\assets\test_assets\NormalsTextureBiasAndScale\Nor
`
and get:

```
C:\Users\ehaines\Documents\_documents\Github\assets\test_assets\NormalsTextureBiasAndScale\NormalsTextureBiasAndScale.usda
python: can't open file 'C:\bin\usdview': [Errno 2] No such file or directory
```

I realized that .bat file wants HFS defined, so I did so:

```
C:\Program Files\Side Effects Software\Houdini 19.5.303\bin>set HFS="C:\Program Files\Side Effects Software\Houdini 19.5.303"

C:\Program Files\Side Effects Software\Houdini 19.5.303\bin>usdview.bat C:\Users\ehaines\Documents\_documents\Github\assets\test_assets\NormalsTextureBiasAndScale\NormalsTextureBiasAndScale.usda
```

Then the pain begins in earnest:

```
C:\Program Files\Side Effects Software\Houdini 19.5.303\bin>hython "C:\Program Files\Side Effects Software\Houdini 19.5.303"/bin/usdview C:\Users\ehaines\Documents\_documents\Github\assets\test_assets\NormalsTextureBiasAndScale\NormalsTextureBiasAndScale.usda
36100: Fatal error: Segmentation fault
-- TRACEBACK BEGIN --
Traceback from hython 19.5.303 (Compiled on windows-x86_64-cl19.29):
CURRENT THREAD 17788
```

Reply:
I'm not sure, sorry, I'm not a regular Houdini user